bin/parseArgs.js

```diff
@@ -0,0 +1,44 @@
+let cleanArgName = (arg) => arg.startsWith('--') ? arg.slice(2) : arg.slice(1)
```

Comment:
Let’s move it to `bin/nanoid.cjs` since we do not use it anywhere

Reply:
Didn't understand you. `cleanArgName` is being used in `bin/parseArgs.js:16` and `bin/parseArgs.js:23`, so in the same file.
Or we can simply omit such sanitizing at all and just use args keys like `parsedArgs['--size']` (and save several bytes). What do you think?

Reply:
I suggested moving all `bin/parseArgs.js` content directly to `bin/nanoid.cjs` (the comment is not for the function,. but for the whole file).

Reply:
Ah. But I test `parseArgs` function separately. In this case, we have to export this function from `bin/nanoid.cjs`. I don't know exactly whether it is OK. Or is it better to test  only real cli exec?

Reply:
Hm. Yes, in this case we can keep a separated file.

Let’s rename it to `bin/utils.js`